Commands/Publishing/GetWikiPageContent.cs

```diff
@@ -7,9 +7,13 @@ namespace SharePointPnP.PowerShell.Commands.Publishing
     [Cmdlet(VerbsCommon.Get, "SPOWikiPageContent")]
     [CmdletHelp("Gets the contents/source of a wiki page",
         Category = CmdletHelpCategory.Publishing)]
+    [CmdletExample(
+        Code = @"PS:> Get-SPOWikiPageContent -PageUrl '/pages/wikipage.aspx'",
```

Comment:
Make the urls clearly server relative. While they are right now, as it can be a page in the root site collection, it's maybe better to use "/sites/demo1/pages/wikipage.aspx", explaining a server relative URL at the same time.